src/ol/proj.js

```diff
@@ -282,8 +283,13 @@ ol.proj.fromLonLat = function(coordinate, opt_projection) {
  * @api
  */
 ol.proj.toLonLat = function(coordinate, opt_projection) {
-  return ol.proj.transform(coordinate,
+  const lonLat = ol.proj.transform(coordinate,
```

Comment:
do we want to start using `const` from es6?

Reply:
Thanks for catching this @fredj.  I'll change this to var for consistency and then we can discuss > es5 separately.

Reply:
Nice catch.

Reply:
When we do introduce features beyond ES5 to the source, we should use Babel to handle transforms to our target browsers when publishing (instead of introducing a dependency on the Closure Compiler).  That said, if we are not targeting IE 10, `const` (and `let`) would make it through untransformed.